.github/workflows/build_docker_image.yml

```diff
@@ -48,6 +48,8 @@ jobs:
         build-args: |
           CMAKE_BUILD_PARALLEL_LEVEL=4
           ASSIMP_VERSION=v5.4.2
+          DRACO_VERSION=1.5.7
+          OCCT_VERSION=V7_8_1
```

Comment:
you do not have this issue ? https://github.com/f3d-app/f3d-superbuild/issues/170

Reply:
hmmm no, maybe because we build as a static lib, or the wasm binary format behaves differently?

Reply:
this is a static build specific issue. Nice if you dont have it.